src/data/faq_de.json

```diff
@@ -515,7 +515,7 @@
                                     "<b>Schritt-für-Schritt-Anleitung: Export aller Zertifikate</b>",
                                     "Um einen Ausdruck aller gespeicherten Digitalen COVID Zertifikate der EU in einem Schritt zu erstellen, gehen Sie bitte folgendermaßen vor: <ul><li>Tippen Sie auf den „Zertifikate“-Reiter in der Corona-Warn-App</li><li>Tippen Sie oben rechts auf das Symbol mit dem nach oben gerichteten Pfeil</li><li>Tippen Sie auf „Weiter“</li><li>Tippen Sie oben rechts auf das Symbol für „Drucken” oder für „Teilen”</li><li>In der folgenden Ansicht können Sie das erstellte PDF-Dokument mit den Standardfunktionen des Betriebssystems drucken, in andere Apps importieren oder auch teilen</li></ul>",
                                     "Zur Veranschaulichung können Sie unter den folgenden Links Screenshots zum Exportieren von Zertifikaten für sowohl iOS als auch Android Geräte finden:<ul><li><a href='../../screenshots/#ios_print_certificate' target='_blank'>iOS Screenshots: Zertifikat drucken</a></li><li><a href='../../screenshots/#android_print_certificate' target='_blank'>Android Screenshots: Zertifikat drucken</a></li></ul>",
-                                    "Siehe auch:<ul><li><a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank'>CWA 2.10: Corona-Warn-App ist bereit, schnell über Auffrischimpfungen zu informieren</a></li><li><a href='../../blog/2022-06-29-cwa-2-24/' target='_blank' >CWA 2.24: Alle Zertifikate in einem PDF-Dokument</a></li><li><a href='../../blog/2022-09-28-cwa-2-27/' target='_blank'>CWA 2.27: Projektteam passt CWA an neues Infektionsschutzgesetz an</a></li><li>FAQ: <a href='#eu_dcc_export_error' target='_blank'>Fehlermeldung 'Druckversion anzeigen nicht möglich'</a></li></ul>"
+                                    "Siehe auch:<ul><li><a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank'>CWA 2.10: Corona-Warn-App ist bereit, schnell über <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a>en zu informieren</a></li><li><a href='../../blog/2022-06-29-cwa-2-24/' target='_blank' >CWA 2.24: Alle Zertifikate in einem PDF-Dokument</a></li><li><a href='../../blog/2022-09-28-cwa-2-27/' target='_blank'>CWA 2.27: Projektteam passt CWA an neues Infektionsschutzgesetz an</a></li><li>FAQ: <a href='#eu_dcc_export_error' target='_blank'>Fehlermeldung 'Druckversion anzeigen nicht möglich'</a></li></ul>"

```

Comment:
This is a link inside a link. I don't think that makes sense!

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!